web/templates/home.html.tmpl

```diff
@@ -4,4 +4,113 @@
     <h1>{{ .Title }}</h1>
     <p>{{ .Paragraph }}</p>
 </div>
+<div class='container'>
+    <div class='card-deck'>
+        <div class='card'>
+            <h5 class='card-header'>Agents</h5>
+            <div class='card-body'>
+                <p class='card-text'>
+                    Though not specifically a view/section, as it is the core component of Trento,
+                    it is important to understand what they are.
+
+                    The agents are the processes that run in each of the nodes that conform
```

Comment:
this (and the others) duplication of text with the docs/ file is a bit unfortunate. how difficult would it be to pull something like https://github.com/gomarkdown/markdown to embed the documentation ?

Reply:
We thought about this, and we already have `gomarkdown/markdown` integrated and available via the `markdown` template function (see #62).
That said, the homepage is likely to change very soon anyway, so we agreed that it would make a simple static HTML page more complicated than it really should.

Reply:
@dirkmueller I already explored this and had a rudimentary version working. But if we want to keep markdown for the scope document it gets quite ugly as one needs to iterate over markdown (or what I did, convert to html and navigate the <h3> tags to render each one of them in a separate card). `JSON` or `yaml` would be better for this but at the expense of losing readability in the document when accessed separately; but IMO this would be overcomplicating this for something that is just meant to be temporal anyways.